CHANGELOG.MD

```diff
@@ -3,7 +3,9 @@ All notable changes to the Chartbeat iOS SDK  will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
-
+## [1.4.3] - 2021-02-18
+### Added
+- Addressed issue with appending slash on viewId in Iphone SDK.
```

Comment:
this should say prepending? right? Also, this change log is only for iphone sdk so no need to mention iphone.

Reply:
Yup prepending is better wording here, updating